tests/posix/eventfd/src/main.c

```diff
@@ -152,7 +152,7 @@ static void test_eventfd_poll_event(void)
 	pfd.fd = fd;
 	pfd.events = POLLIN;
 
-	ret = poll(&pfd, 1, K_SECONDS(3));
+	ret = poll(&pfd, 1, 3000);
```

Comment:
`3 * MSEC_PER_SEC` ?

Reply:
Nope, just time in milliseconds, as accepted by POSIX poll().